back/src/routes/airtable.ts

```diff
@@ -91,43 +89,73 @@ airtableRouter.post("/getInfo", function (req, res) {
       contactMethod: "",
       paymentMethod: "",
     };
-    console.log("what is the req.body: " + JSON.stringify(req.body))
     base("Authentication")
       .select({ filterByFormula: `{Username} = "${userName}"` })
       .firstPage((err, records) => {
         if (err) console.error(err);
         if (records.length != 1)
```

Comment:
Wouldn't we want this to be ```records.length < 1```?

Reply:
Well yeah, this code was copied from the last team. How would you do it?